### PR TITLE
Update readme for dummies & docker compose 

### PR DESCRIPTION
**Pull request by vegber**

### CODE REVIEW

**Comment by vegber** [comment]
@Celsiusss  please view my work

**Review by Celsiusss — COMMENTED** [review]
Instead of docker-compose containing postgres, why not just include
```
docker run --name postgres -p "5432:5432" -e POSTGRES_PASSWORD=abc123 -e POSTGRES_DB=wakiewakie postgres
```
in the readme?

File: Dockerfile
Comment (Celsiusss) [inline Dockerfile:1]: empty file??

Comment (vegber) [reply Dockerfile:1]: A dockerfille could be nice to have, as for future execution of the program we could include: 

```sh 
# Use an official Python runtime as a parent image
FROM python:3.11

# Set the working directory in the container
WORKDIR /usr/src/app

# Copy the current directory contents into the container at /usr/src/app
COPY . .

# Install any needed packages specified in requirements.txt
RUN pip install --no-cache-dir -r requirements.txt

# Make port 80 available to the world outside this container
EXPOSE 8000

# Run the application when the container launches
CMD ["uvicorn", "wakiewakie.main:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
```

as an example

**Comment by vegber** [comment]
> Instead of docker-compose containing postgres, why not just include
> 
> ```
> docker run --name postgres -p "5432:5432" -e POSTGRES_PASSWORD=abc123 -e POSTGRES_DB=wakiewakie postgres
> ```
> 
> in the readme?

Becuase it looks better and is easier to understand

**Review by Celsiusss — APPROVED** [review]
lgtm

**Pull request merged**